src/config/networks.ts

```diff
@@ -58,7 +58,8 @@ export const NetworkList: Networks = {
         dark: '#9fccaa',
       },
     },
-    subscanEndpoint: 'https://subscan-mainnet.creditcoin.network/',
+    subscanEndpoint: 'https://subscan-mainnet.creditcoin.network',
+    subscanUrl: 'https://creditcoin.subscan.io',
```

Comment:
@Sanggon-Kim naming of this is mildly confusing but if I understand it correctly `subscanUrl` is the value for using Subscan as a service, while `subscanEndpoint` is the value for self-hosted Subscan fork. Is this correct ? 

Q2: if we are switching over to a hosted Susbcan service shouldn't references to the old `subscanEndpoint` variable be removed then ?  I don't see this to be the case.